data/json/professions.json

```diff
@@ -4,13 +4,13 @@
             "ident": "bionic_prepper",
             "name": "Bionic Prepper",
             "description": "You knew the end was coming.  You augmented yourself with some basic bionics and got additional survival training.  Now the end has come, and it is time to see if your efforts have paid off.",
-            "points": 8,
+            "points": 6,
             "CBMs":[
                 "bio_torsionratchet",
-                "bio_power_storage",
-                "bio_tools",
+                "bio_power_storage_mkII",
                 "bio_sunglasses",
-                "bio_blindfold"
```

Comment:
I didn't check it, but does anti-glare bionic correctly add the blindfold if added at start?
It would be a bug if any character had one but not the other.